README.rst

```diff
@@ -13,7 +13,7 @@
 jsonargparse
 ============
 
-https://github.com/omni-us/jsonargparse/
```

Comment:
When clicking on it from readthedocs, it would take you to github. But if on github, it would do nothing.